DEVELOPERS.md

```diff
@@ -0,0 +1,175 @@
+![Circus Train BigQuery.](circus-train-bigquery.png "Moving data from Google BigQuery to Hive.")
+
+## Overview 
+Circus Train BigQuery is an extension of Circus Train which allows tables in Google BigQuery to be replicated.
+
+This file contains a collection of notes put together by developers who have worked on BigQuery to provide some explanation to the code. This file is not completely exhaustive of all the inner workings of the project, so do feel free to add more information or detail. 
+
+
+## README.md
+
+First and foremost, its worth having a read through the Circus Train [README.md](https://github.com/HotelsDotCom/circus-train) file and the Circus Train BigQuery [README.md](https://github.com/HotelsDotCom/circus-train-bigquery/blob/master/README.md). These are pretty extensive guides containing a lot of info on the projects, including how to run each of them and the different configurations which can be used. 
+
+It may also be useful to read through the DEVELOPERS.md file for Circus Train. 
```

Comment:
Make this a link too (you can figure out what the location should be, but we should merge that other PR first).

Reply:
Yeah I will do 👍